experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -316,6 +317,49 @@ describe("SharedTree", () => {
 		});
 	});
 
+	it("Uses handles in the summary when there are no schema changes since the last summary.", async () => {
```

Comment:
This logic isn't very simple or an established pattern: we should have more test coverage for it.

One way to organize some interesting test cases:

```typescript

describe("SharedTree", () => {
	describe("schema index summarization", () => {
		describe("incrementally reuses previous blobs", () => {
			it("on a client which never uploaded a blob");
			it("on a client which uploaded a blob")
			// this concern is orthogonal to the two above, but important
			it("if there are changes unrelated to schemaIndex");
		});

		describe("uploads new schema data", () => {
			it("without incremental summary context");
			it("when it has changed since the last summary");
		});
	});
})
```

Reply:
updated unit tests with suggested test cases